### DISCUSSION TITLE
Definition of j and i

### DISCUSSION THREAD

**Discussion by Rasmussoe123** [Eduap-com/WordMat#183] [Q&A]
Im trying to define j as i, when doing an equation. Is that possible?
10+i*15
but i want to be able to do it as
10+j*15
Thanks

**Comment by Diego-PB** [comment]
If you want to use "j" instead of "i" to represent the imaginary unit, you can do so. In many contexts, "j" is used instead of "i" to avoid confusion with notation. So, your expression would look like this:
10+j15
It's the same as writing 10+15i

But I'm not sure I completely understand your problem, so maybe my answer isn't correct.
